utils/docify/docify/cli.py

```diff
@@ -0,0 +1,95 @@
+"""
+Docify.
+
+Usage:
+  docify [--verbose] <configfile> [<stubpath>]
+  docify -h | --help
+  docify --version
+
+Options:
+  --verbose     Print out details of what docify is doing.
+  -h --help     Show this screen.
+  --version     Show version.
+
+If stubpath is not specified the current working directory 
+will be assumed.
+
+The config file is a CSV file that has lines of the form:
+
+    package,stub_file_path,classname,methodname
+
+If classname is empty, methodname is a top-level function name.
+The stub_file_path paths should be relative to stubpath.
+
+The stub file specified by stub_file_path will be patched with 
+the docstring of classname.methodname, which will be extracted
+by introspection from the specified package.
+
+For example:
+
+    pandas,core/series.pyi,Series,groupby
+
+"""
+import csv
+import importlib
+import inspect
+import os
+import sys
+from docopt import docopt
+import docify
+
+
+def main():
+    arguments = docopt(__doc__, version='docify 0.1')
+    configfile = arguments['<configfile>']
+    stubpath = arguments['<stubpath>']# if '<stubpath>' in arguments else '.'
+    if stubpath is None:
+        stubpath = '.'
+    verbose = arguments['--verbose']
+    with open(configfile) as f:
+        patches = csv.reader(f)
+
+        for patch in patches:
+            if not patch:
+                break
+            pkg, stub, class_, method = patch
+            try:
+                if pkg in sys.modules:
+                    package = sys.modules[pkg]
+                else:
+                    package = importlib.import_module(pkg)
+
+                x = package
```

Comment:
Give x a more informative name

Reply:
x here could be a package, module, class, or function/method, as we go through the hierarchy. So it is a true 'variable', and I deliberately chose x. But open to suggestions if you think you have a better name.

Reply:
Maybe something like currentObject? Or currentScope?

Reply:
I went for obj; camel cased vars are not idiomatic in Python and I think obj is suggestive enough.